spaces/domains.json

```
@@ -228,5 +228,6 @@
   "vote.steakhouse.finance": "vote-steakhouse.eth",
   "vote.flashstake.io": "flashcommunity.eth",
   "vote.fenixtoken.net": "fenixtoken.eth",
+  "vote.supershiba.exchange": "supershiba.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.supershiba.exchange": "supershiba.eth",
```